### ISSUE TITLE
Property 'connectedCallback' does not exist on type ApolloQuery [typescript]

### ISSUE THREAD

**Issue by padzikm** [apollo-elements/apollo-elements#37]
## Package
`@apollo-elements/lit-apollo`

## Description
Transpilation produces error saying that Property 'connectedCallback' does not exist on type ApolloQuery. Indeed:
apollo-query.d.ts:
```
import { ApolloQuery as ApolloQueryBase } from "@apollo-elements/mixins/apollo-mutation-mixin";
...
```
query inherits from mutation, not from from query. 
Furthermore apollo-element-mixin.d.ts:
```typescript
...
export declare class ApolloElement<TBase, TData> extends HTMLElement {
...
```
ApolloElement inherits fromt HTMLElement, not from LitElement, which is TBase

## Steps To Reproduce
test.ts:
```typescript
import {ApolloQuery} from '@apollo-elements/lit-apollo'

class Test extends ApolloQuery<any, any>{
   connectedCallback(){
      super.connectedCallback()
   }
}
```

## Expected Behavior
No typescript errors while transpiling above code

**Comment by bennypowers** [comment]
I'm not a TS expert. I mostly added the typings files for editor support. I'm more than happy to take a PR for it though, especially with tests.

**Comment by bennypowers** [comment]
closed by #46 
https://www.npmjs.com/package/@apollo-elements/lit-apollo/v/1.0.2
